modules/swagger-codegen/src/main/resources/JavaSpring/beanValidation.mustache

```diff
@@ -1,5 +1,6 @@
 {{#required}}
   @NotNull
-{{/required}}
+{{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}
```

Comment:
There is the rub, the Hibernate part. I'd have to retest all my beans again to determine if there will be impact with this approach. I do not have any Hibernate backed beans.

Thanks for the information.